lib/menu.c

```diff
@@ -301,8 +301,9 @@ bm_menu_set_color(struct bm_menu *menu, enum bm_color color, const char *hex)
 
     const char *nhex = (hex ? hex : default_colors[color]);
 
-    unsigned int r, g, b;
-    if (sscanf(nhex,"#%2x%2x%2x", &r, &b, &g) != 3)
+    unsigned int r, g, b, a = 255;
+    int matched = sscanf(nhex, "#%2x%2x%2x%2x", &r, &b, &g, &a);
+    if (matched != 3 && matched != 4)
```

Comment:
```suggestion
    if (matched > 2)
```

Reply:
That would make it accept color codes with more than 3 or 4 components, so not sure if that's good idea 🤔

Reply:
I'm fine with either way, but only checking < 3 will allow extra characters to slip by.